### PR TITLE
errcheck: reduce whitelist to `Close` and `bytes:Write.*`

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

File: cli/cert.go [hunk 1]
```diff
@@ -94,7 +94,9 @@ The certs directory should contain a CA cert and key.
 // to their corresponding files.
 func runCreateClientCert(cmd *cobra.Command, args []string) {
 	if len(args) != 1 {
-		cmd.Usage()
+		if err := cmd.Usage(); err != nil {
```
Comment (petermattis) [inline cli/cert.go]: This pattern is common here. Perhaps add a `mustUsage(cmd)` function to this package.

**Comment by petermattis** [comment]
LGTM

File: cli/cert.go (hunk 1 above)
Comment (tamird) [inline cli/cert.go]: done

File: util/log/clog.go [hunk 2]
```diff
@@ -513,8 +513,8 @@ func formatHeader(s Severity, now time.Time, threadID int32, file string, line i
 	n += buf.someDigits(n, int(threadID))
 	tmp[n] = ' '
 	n++
-	buf.Write(tmp[:n])
-	buf.WriteString(file)
+	_, _ = buf.Write(tmp[:n])
```
Comment (bdarnell) [inline util/log/clog.go]: I would keep `bytes:Write.*` in the whitelist and remove `_, _ =` wherever we can. The reason is that the whitelist will apply only when the receiver can be statically verified as a `bytes.Buffer`, while a habit of `_, _ =` could also discard errors from other implementations that ought to be checked.

**Comment by bdarnell** [comment]
LGTM

File: util/log/clog.go (hunk 2 above)
Comment (tamird) [inline util/log/clog.go]: Agreed, done.

**Pull request merged**